fails to build with ghc-9.6

```
       Building library for list-transformer-1.0.9..
       [1 of 1] Compiling List.Transformer
       
       /var/stackage/work/unpack-dir/unpacked/list-transformer-1.0.9-8aa9799ac936ad31c9fa174617ac8181e5fc376e9c6a6d6aae942d296233d740/src/List/Transformer.hs:821:58: error: [GHC-39999]
           • Could not deduce ‘Monad (ZipListT m)’
               arising from the head of a quantified constraint
               arising from the 'deriving' clause of a data type declaration
             from the context: Monad m
               bound by a quantified context at src/List/Transformer.hs:821:58-67
           • When deriving the instance for (MonadTrans ZipListT)
           |
       821 |   deriving (Functor, Alternative, Foldable, Traversable, MonadTrans, Floating, Fractional, Num, Semigroup, Monoid)
           |                                                          ^^^^^^^^^^
```

--- a/src/List/Transformer.hs
+++ b/src/List/Transformer.hs
@@ -818,7 +818,7 @@ instance MFunctor Step where
 -- that expects "an Applicative instance", written to be polymorphic over
 -- all Applicatives.
 newtype ZipListT m a = ZipListT { getZipListT :: ListT m a }
-  deriving (Functor, Alternative, Foldable, Traversable, MonadTrans, Floating, Fractional, Num, Semigroup, Monoid)
+  deriving (Functor, Alternative, Foldable, Traversable, Floating, Fractional, Num, Semigroup, Monoid)
 
 instance Monad m => Applicative (ZipListT m) where
     pure x = ZipListT go

